samples/http_2.js

```diff
@@ -1,9 +1,16 @@
 import http from "k6/http";
 import { check } from "k6";
 
+if (__VU == 1) {
+    http.setResponseCallback(http.expectedStatuses(200))
+} else {
+    http.setResponseCallback(http.expectedStatuses(300))
+}
 export default function () {
+    console.log(JSON.stringify(http));
```

Comment:
This seems like some debug code that snuck in

Reply:
yeah ... I apparently once didn't hit `n` when `git add -p`-ing :facepalm: